### ISSUE TITLE
Learning rate and overfitting

### ISSUE THREAD

**Issue by Seohyeong** [Seohyeong/ShroomAI#1]
<img width="977" alt="Screenshot 2024-10-12 at 6 15 09 PM" src="https://github.com/user-attachments/assets/ec0b06e2-24b1-4fa5-9940-ec84d4768bdf">

Increasing the learning rate beyond a certain point can lead to overfitting
